phys2bids/interfaces/acq.py

```diff
@@ -21,23 +21,29 @@ def populate_phys_input(filename, chtrig):
     filename: str
         path to the txt labchart file
     chtrig : int
-        index of trigger channel
+        index of trigger channel.
+        !!! ATTENTION: IT'S MEANT TO REPRESENT AN INDEX STARTING FROM 1 !!!
 
     Returns
     -------
     BlueprintInput
 
+    Note
+    ----
+    chtrig is not a python index - instead, it's human readable.
+    This is handy because when initialising the class, it gets initiliased
+    with a channel more at the beginning - that is already taken into account!
```

Comment:
```suggestion
    chtrig is not a 0-based Python index - instead, it's human readable (i.e., 1-based).
    This is handy because, when initialising the class, a new channel corresponding
    to time is added at the beginning - that is already taken into account!
```

Reply:
You don't need to accept this suggestion. I think it would be sufficient to just clarify that the new channel reflects time. Also, `initialised` was misspelled as `initiliased` (which might have gotten buried in the suggestion).